xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/vue/LivedataDropdownMenu.vue

```diff
@@ -103,6 +103,13 @@
         </a>
       </li>
 
+      <li v-for="extensionPanel in logic.extensionPanels" :key="extensionPanel.id">
+        <a href="#" @click.prevent="logic.uniqueArrayToggle(logic.openedPanels, extensionPanel.id)">
+          <XWikiIcon :icon-descriptor="{name: extensionPanel.icon}"/>
+          {{ extensionPanel.name }}
+        </a>
+      </li>
+
```

Comment:
I wonder if it's interesting to visually separate the 'standard' panels from the one provided by extension.

Reply:
I don't think that's interesting from a user's point of view as in general we don't separate between output that has been produced in extensions vs. in core. In fact, I think it could be more interesting to let extensions add panels between the existing ones.

Reply:
Sounds good. Definitely for a next iteration. Having the extension panels at the end of the list is fine for now.